src/lightbox/LightboxActionSheet.js

```diff
@@ -1,5 +1,5 @@
 /* @flow */
-import type { Auth } from '../types';
+import type { Auth, TranslateStringType } from '../types';
```

Comment:
We are not naming types `*Type`. Remove that part.
Also, try to think of a more descriptive name. Maybe `StringTranslator` or even `TextTranslator` (as 'text' is used in 'getText')

Reply:
Done